logstash-core/lib/logstash/environment.rb

```diff
@@ -34,6 +34,7 @@ module Environment
   end
 
   [
+            Setting::String.new("on_superuser", "WARN"),
```

Comment:
Similarly to other settings:

```
            Setting::String.new("log.level", "info", true, ["fatal", "error", "warn", "debug", "info", "trace"]),
```

 we can list the acceptable values in the setting definition:

```suggestion
            Setting::String.new("on_superuser", "WARN", true, ["ALLOW", "WARN", "BLOCK"]),
```